### ISSUE TITLE
docs for to_int_unchecked should mention the safe version 

### ISSUE THREAD

**Issue by Lokathor** [rust-lang/portable-simd#244]
The docs for to_int_unchecked should mention that round is the safe version of the function.

**Comment by programmerjake** [comment]
uuh, round isn't the safe version of to_int_unchecked:
https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=a68ddebbf0e80e37bf027ff5d2738350

```
[src/main.rs:2] unsafe { 1.7f32.to_int_unchecked() } = 1
[src/main.rs:3] 1.7f32.round() = 2.0
[src/main.rs:4] 1.7f32 as i32 = 1
```

**Comment by Lokathor** [comment]
ah, so it's a truncate

**Comment by workingjubilee** [comment]
Yes, roughly following [`cvttps2dq`](https://www.felixcloutier.com/x86/cvttps2dq) for the obvious cast of `to_int_unchecked::<i32>(self) -> i32`. `Simd::cast()` will exhibit the correct safe behavior most of the time. It is completely reasonable for us to mention the safe function.
